--- arm.cpp	2013-08-07 22:06:10.824191526 -0300
+++ arm.cpp.new	2013-08-07 22:23:25.994164156 -0300
@@ -28,12 +28,12 @@
   for (;;) {
 
   bhv_pc = ac_pc;
-  if( bhv_pc >= APP_MEM->get_size()){
-    cerr << "ArchC: Address out of bounds (pc=0x" << hex << bhv_pc << ")." << endl;
-    stop();
-    return;
-  }
-  else {
+//  if( bhv_pc >= APP_MEM->get_size()){
+//    cerr << "ArchC: Address out of bounds (pc=0x" << hex << bhv_pc << ")." << endl;
+//    stop();
+//    return;
+//  }
+//  else {
     if( start_up ){
       decode_pc = ac_pc;
       start_up=0;
@@ -505,7 +505,7 @@
     delete instr_vec;
     if ((!ac_wait_sig) && (!ac_annul_sig)) ac_instr_counter+=1;
     ac_annul_sig = 0;
-  }
+//  }
 //!Updating Regs for behavioral simulation.
   if(!ac_wait_sig){
     bhv_pc = ac_pc;
